ACMILabs/interactive-label
# #128 Add large text and zoomable images

Resolves #128

Let's make some accessibility additions - large text and zoomable images.

### Acceptance Criteria
- [x] Add button to increase font-size by `1.5x`
- [x] Add ability to zoom images

### Relevant design files
* [Figma](https://www.figma.com/file/r8dJf66KcjkRONvMTqd8rF/VE---Gallery?node-id=847%3A15276)

### Testing instructions
1. Set your `config.env` to the settings below (or another interactive label of your choice)2. 
1. Load it up: `cd development` and `docker-compose up`: http://localhost:8081
1. Note that the large text works as expected
1. Note that the pinch/zoom/pan images works as expected

```env
# config.env
XOS_API_ENDPOINT=https://xos-stg.acmi.net.au/api/
AUTH_TOKEN=37b3c9e8ec9d10ce668dbd6e27fd259a14fc2cee
# FSF
XOS_PLAYLIST_ID=136
